Variable.csv

```diff
@@ -0,0 +1,150 @@
+tableName,variableName,key,variableType,variableDesc
+Sample,sampleID,Primary Key,string,Unique identification for sample. Suggestion:�siteID-date-index.
+Sample,siteID,Foreign key,string,Links with the Site table to describe the location of sampling.
+Sample,dateTime,,datetime,"for grab samples this is the�date, time and timezone�the sample was taken."
+Sample,dateTimeStart,,datetime,"For integrated time averaged samples this is the�date, time and timezone�the sample was started being taken."
+Sample,dateTimeEnd,,datetime,"For integrated time average samples this is the�date, time and timezone�the sample was finished being taken."
+Sample,type,,category,Type of sample.
+Sample,typeOther,,string,Description for other type of sample not listed in
+Sample,collection,,category,Method used to collect the data.
+Sample,collectionOther,,string,Description for other type of method not listed in�collection.
+Sample,preTreatment,,boolean,Was the sample chemically treated in anyway with the addition of stabilizers or other?
+Sample,preTreatmentDescription,,string,If�preTreatment�then describe the treatment that was performed.
+Sample,pooled,,boolean,"Is this a pooled sample, and therefore composed of multiple child samples obtained at different sites?"
+Sample,children,,string,If this is a sample with many smaller samples either because of pooling or sub-sampling this indicates�a comma separated list of child sampleID's.
+Sample,parent�,,string,If this sample has been pooled into one big sample for analysis this indicates the�sampleID of the larger pooled sample.
```

Comment:
Looks like there are a bunch of special characters like, 'parent�' that are suppose to be there??

Reply:
thanks @DougManuel looking into it